src/main/java/frc/robot/commands/OrganizeIndexCMD.java

```diff
@@ -25,14 +25,7 @@ public void initialize() {
 
   }
```

Comment:
If there's no `execute()` and this command can't do anything, why is it still here?